__tests__/unit/orm/models-test.js

```diff
@@ -1,69 +1,145 @@
+        { isPolymorphic = false } = {}
+      ) {
+        expect(model.__proto__[name].constructor.name).toBe(type);
+        expect(model.__proto__[name].modelName).toBe(modelName);
+        expect(model.__proto__[name].opts.polymorphic).toBe(isPolymorphic);
+      }
+
+      // eslint-disable-next-line jest/expect-expect
+      it("belongs to test category", function () {
+        testRelationship("belongsToField", "BelongsTo", "test-category");
+      });
+
+      // eslint-disable-next-line jest/expect-expect
+      it("has many test options", function () {
+        testRelationship("hasManyField", "HasMany", "test-option");
+      });
+
+      // eslint-disable-next-line jest/expect-expect
+      it("belongs to test interface", function () {
+        testRelationship("interfaceField", "BelongsTo", "test-interface", {
+          isPolymorphic: true,
+        });
+      });
+
+      // eslint-disable-next-line jest/expect-expect
+      it("has many test relay nodes", function () {
+        testRelationship("relayConnectionField", "HasMany", "test-relay-node");
+      });
+
+      // eslint-disable-next-line jest/expect-expect
+      it("has many test unions", function () {
+        testRelationship("unionField", "HasMany", "test-union", {
+          isPolymorphic: true,
+        });
+      });
+    });
   });
 
-  describe("TestObject relationships", function () {
-    const model = new registeredModels.TestObject({}, "TestObject");
-
-    function testRelationship(
-      name,
-      type,
-      modelName,
-      { isPolymorphic = false } = {}
-    ) {
-      expect(model.__proto__[name].constructor.name).toBe(type);
-      expect(model.__proto__[name].modelName).toBe(modelName);
-      expect(model.__proto__[name].opts.polymorphic).toBe(isPolymorphic);
+  describe("createModels", function () {
+    const models = createModels({ graphQLSchema });
+
+    function findModel(name) {
+      return models.filter((model) => {
+        return model.name == name;
+      })[0];
     }
 
-    // eslint-disable-next-line jest/expect-expect
-    it("belongs to test category", function () {
-      testRelationship("belongsToField", "BelongsTo", "test-category");
+    it("does not register a model for the mutation type", function () {
+      expect(findModel("Mutation")).toBeUndefined();
     });
 
-    // eslint-disable-next-line jest/expect-expect
-    it("has many test options", function () {
-      testRelationship("hasManyField", "HasMany", "test-option");
+    it("does not register a model for the query type", function () {
+      expect(findModel("Query")).toBeUndefined();
     });
 
-    // eslint-disable-next-line jest/expect-expect
-    it("belongs to test interface", function () {
-      testRelationship("interfaceField", "BelongsTo", "test-interface", {
-        isPolymorphic: true,
-      });
+    it("does not register a model for the subscription type", function () {
+      expect(findModel("Subscription")).toBeUndefined();
```

Comment:
Small ask here... Can the test descriptions be changed to match the context? Can we say `it("does not create a model...")` instead of "register", to add clarity?